src/dynamics/ergodic/ergodic.lean

```diff
@@ -68,6 +68,45 @@ lemma of_iterate (n : ℕ) (hf : pre_ergodic (f^[n]) μ) : pre_ergodic f μ :=
 
 end pre_ergodic
 
+namespace measure_theory.measure_preserving
+
+variables {β : Type*} {m' : measurable_space β} {μ' : measure β} {s' : set β} {g : α → β}
+
+lemma pre_ergodic_of_pre_ergodic (hg : measure_preserving g μ μ')
```

Comment:
The name is pretty obscure. `pre_ergodic_of_pre_ergodic_conjugate`? It's not exactly true, but better I think.

Reply:
Agreed, this is better.